ilan/README.md

```diff
@@ -0,0 +1,69 @@
+# Building an IBM HTTP Server ILAN image from binaries
+
+An IBM HTTP Server ILAN image can be built by obtaining the following binaries:
```

Comment:
Can we actually say what these binaries are to help people find them and know exactly what files they are expected to use?